utils/bump_cwl_version.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python3
+
+import argparse
+import glob
+import logging
+import yaml
+
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def tools_list(tools_dir):
+  log.debug(f'tools_dir passed to tools_list={tools_dir}')
+  glob_pattern = f'{tools_dir}/*.cwl'
+  log.debug(f'glob_pattern ={glob_pattern}')
+  return glob.glob(glob_pattern)
+
+
+def read_tool(path):
+  with open(path) as file:
+    tool = yaml.load(file, Loader=yaml.FullLoader)
+  return tool
+
+
+def edit_tool(tool, new_version):
+  # this is dependent on cwl syntax -- you'll need to change this
+  # if you use the list style for hints, or put your DockerRequirement
+  # under 'requirements' instead of 'hints'
```

Comment:
This does make me a little bit nervous, but we can manage it if it's a requirement for any tools that we are sharing

Reply:
I can work on this function supporting more variation.

Reply:
@wpoehlm I have made the bump script tolerate a lot more syntactic variation in commit [c573856](https://github.com/Sage-Bionetworks-Workflows/dockstore-tool-template/pull/4/commits/c573856a1960ac216f10262ac2aa1eecda3f70d2)